### ISSUE TITLE
[bug]: Install Error After Power Cut

### ISSUE THREAD

**Issue by ZenRyuujin** [invoke-ai/InvokeAI#2410]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### OS

Windows

### GPU

cuda

### What happened?

During initial installation, I had a power cut before installation could finish. The power cut happened after installing all requirements during the Model download phase. On attempting to resume the installation, I started getting an error (see below).

Building wheels for collected packages: InvokeAI
  Building wheel for InvokeAI (setup.py) ... done
  Created wheel for InvokeAI: filename=InvokeAI-2.2.5-py3-none-any.whl size=2049642 sha256=b8c4b672da0c37edfdd56521fab9cbe67f3a27e6e2e19fadca71f70121011ba7
  Stored in directory: C:\Users\username\AppData\Local\Temp\pip-ephem-wheel-cache-x9y55u0d\wheels\05\d1\14\7329c9dd2ba9b671a7a0c88268926239feea8ba165fd4bf269
Successfully built InvokeAI
Installing collected packages: InvokeAI
Successfully installed InvokeAI-2.2.5
Installation successful.
.\templates\rootdir\configs\INITIAL_MODELS.yaml
.\templates\rootdir\configs\models.yaml.example
.\templates\rootdir\configs\sd-concepts.txt
.\templates\rootdir\configs\stable-diffusion\v1-finetune.yaml
.\templates\rootdir\configs\stable-diffusion\v1-finetune_style.yaml
.\templates\rootdir\configs\stable-diffusion\v1-inference.yaml
.\templates\rootdir\configs\stable-diffusion\v1-inpainting-inference.yaml
.\templates\rootdir\configs\stable-diffusion\v1-m1-finetune.yaml
8 File(s) copied
Loading Python libraries...

Traceback (most recent call last):
  File "d:\invokeai\.venv\Scripts\configure_invokeai.py", line 24, in <module>
    from transformers import CLIPTokenizer, CLIPTextModel
  File "d:\invokeai\.venv\lib\site-packages\transformers\__init__.py", line 30, in <module>
    from . import dependency_versions_check
  File "d:\invokeai\.venv\lib\site-packages\transformers\dependency_versions_check.py", line 17, in <module>
    from .utils.versions import require_version, require_version_core
  File "d:\invokeai\.venv\lib\site-packages\transformers\utils\__init__.py", line 59, in <module>
    from .hub import (
  File "d:\invokeai\.venv\lib\site-packages\transformers\utils\hub.py", line 1087, in <module>
    cache_version = int(f.read())
ValueError: invalid literal for int() with base 10: '\x00'
Configuration failed. See above for error messages and check https://invoke-ai.github.io/InvokeAI/installation/INSTALL_AUTOMATED/#troubleshooting for potential solutions.
The installer will exit now.
Press any key to continue . . .

I have attempted to delete the whole folder and start over, continue the installation from where it is, and delete and reinstall only the specific module in the error. None of the above have worked.
I'm out of ideas on how to fix this.

**Comment by ZenRyuujin** [comment]
I "fixed" my own problem but am not happy with doing it this way. I had to change the code in transformers to get past the error.

By changing the site-package at the position indicated below,
-----
yadayada\.venv\lib\site-packages\transformers\utils\hub.py", line 1087, in
cache_version = int(f.read())
----
to the following,:
----
cache_version_file = os.path.join(TRANSFORMERS_CACHE, "version.txt")
if not os.path.isfile(cache_version_file):
    cache_version = 0
else:
    with open(cache_version_file) as f:
        **try:**
            cache_version = int(f.read())
        **except ValueError as e:
            print("This is where I was getting a weird value error. "
                  "Converting the odd value to 0 base 10 instead of 0 base 16")
            cache_version = 0**
----
I caught the error and got past the point where it broke the install and it finished running.
Code in Bold is what I added.

tl;dr
Somewhere, there is a hex zero being used where there should be a decimal zero and it causes problems sometimes.

**Comment by lstein** [comment]
Sorry to be responding to this late, but you should have been able to fix the problem by removing the `invokeai` directory and its `.venv` subdirectory. `.venv` contains all the libraries that are installed, and apparently one of them became corrupted during your power cut.
